core/embed/rust/src/protobuf/obj.rs

```diff
@@ -150,11 +152,12 @@ pub struct MsgDefObj {
 }
 
 impl MsgDefObj {
-    pub fn alloc(def: MsgDef) -> Gc<Self> {
+    pub fn alloc(def: MsgDef) -> Result<Gc<Self>, Error> {
         Gc::new(Self {
             base: Self::obj_type().to_base(),
             def,
         })
+        .map_err(Into::into)
```

Comment:
I'd rather write it with `?` and a `let`. I suppose it would also be a bit more consistent with the rest of the codebase. Small nit.

Reply:
fixed during rebase